### PR TITLE
Adding TestStand .seq example for NI-SCOPE.

### PR DESCRIPTION
**Pull request by DelpireNI**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Creates an example TestStand sequence file for our NI-SCOPE Acquire Waveform Example.
image

### Why should this Pull Request be merged?

We want example sequences for all of our examples.
[#AB2540544](https://dev.azure.com/ni/DevCentral/_backlogs/backlog/ASW%20MeasurementLink/Features?showParents=false&workitem=2540544)

### What testing has been done?

TestStand Sequences runs successfully.

### CODE REVIEW

File: Source/Example Measurements/NI-SCOPE Acquire Waveform/NI-SCOPE Acquire Waveform/TestStand Code Modules/Create and Register Sessions.vi
Comment (dixonjoel) [inline Source/Example Measurements/NI-SCOPE Acquire Waveform/NI-SCOPE Acquire Waveform/TestStand Code Modules/Create and Register Sessions.vi:1]: Copy-paste error.
image

Comment (dixonjoel) [reply Source/Example Measurements/NI-SCOPE Acquire Waveform/NI-SCOPE Acquire Waveform/TestStand Code Modules/Create and Register Sessions.vi:1]: Consider how this should have affected your manual testing and what additional tests you should run to verify this fix.

File: Source/Example Measurements/NI-SCOPE Acquire Waveform/NI-SCOPE Acquire Waveform/TestStand Code Modules/Unregister and Destroy Sessions.vi
Comment (dixonjoel) [inline Source/Example Measurements/NI-SCOPE Acquire Waveform/NI-SCOPE Acquire Waveform/TestStand Code Modules/Unregister and Destroy Sessions.vi:1]: Update this comment for SCOPE
image

**Review by dixonjoel — APPROVED** [review]

File: Source/Example Measurements/NI-SCOPE Acquire Waveform/NI-SCOPE Acquire Waveform/TestStand Code Modules/Create and Register Sessions.vi
Comment (DelpireNI) [reply Source/Example Measurements/NI-SCOPE Acquire Waveform/NI-SCOPE Acquire Waveform/TestStand Code Modules/Create and Register Sessions.vi:1]: Yes this is strange. I guess this is being ignored.

File: Source/Example Measurements/NI-SCOPE Acquire Waveform/NI-SCOPE Acquire Waveform/TestStand Code Modules/Unregister and Destroy Sessions.vi
Comment (DelpireNI) [reply Source/Example Measurements/NI-SCOPE Acquire Waveform/NI-SCOPE Acquire Waveform/TestStand Code Modules/Unregister and Destroy Sessions.vi:1]: Fixed

**Review by jasonmreding — APPROVED** [review]

**Pull request merged**